http.c

```diff
@@ -1160,15 +1160,21 @@ Datum http_request(PG_FUNCTION_ARGS)
 		content_text = DatumGetTextP(values[REQ_CONTENT]);
 		content_size = VARSIZE_ANY_EXHDR(content_text);
 
-		if ( method == HTTP_GET || method == HTTP_POST )
+		if ( method == HTTP_GET || method == HTTP_POST || method == HTTP_DELETE )
```

Comment:
You've tested this and it works for you? Want to make sure as the use of CURLOPT_POSTFIELDS is not guaranteed over all commands, hence the special handling below for PUT/PATCH.

Reply:
I tried it yesterday with some examples where it worked like a charm. However, I'm not sure if it varies from execution to execution, so that one time it works and the other time it doesn't?